README.md

```diff
@@ -1 +1,35 @@
 # profoilio-header-headline
+![image](https://user-images.githubusercontent.com/43178495/129184003-fce2ee99-75d6-4215-b45c-0d33d13fb9a5.png)
+
+prepare portfolio website
+# Additional description about the project and its features.
+
+This project Is for setting header and headline dsectioin to portfolio website using sementic HTML tags and clean style file
```

Comment:
- [ ] You have a spelling error in this line

Reply:
Yeah I have fixed that please check it again

Reply:
fixed